### PR TITLE
fix upload-file command when the file is summary report

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17758

## Description
The command servicenow-upload-file failed when using the file generated with generateSummaryReport script.

### CODE REVIEW

File: Integrations/ServiceNow/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+fix servicenow-upload-file command when the file is summary report
```
Comment (liorblob) [inline Integrations/ServiceNow/CHANGELOG.md]: InfoFile is not just for summary report, just write info file instead.

File: Integrations/ServiceNow/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix `servicenow-upload-file` command when the file is summary info file.
```
Comment (liorblob) [inline Integrations/ServiceNow/CHANGELOG.md]: Remove `summary` :)

**Comment by adi88d** [comment]
@kirbles19 Please review the RN here.

**Comment by kirbles19** [comment]
@adi88d Done.

**Pull request merged**